src/Symfony/Component/Security/Http/FirewallMapInterface.php

```diff
@@ -21,16 +21,19 @@
 interface FirewallMapInterface
 {
     /**
-     * Returns the authentication listeners, and the exception listener to use
-     * for the given request.
+     * Returns the authentication listeners, the exception listener and the logout
```

Comment:
just chatted a bit with @nicolas-grekas, let's revert the interface changes now (changelog update not needed anymore), we need to change this on 4.1 with proper deprecation.

Reply:
agreed: should be reverted and reintroduced in 4.1 with a deprecation notice

Reply:
okay done